Feature/router

knees4bees opened the pull request:
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [ ] Fix
- [x] Feature (to add router)
- [x] Refactor (to add planet images rather than icons on detail view)
- [x] Style (very minor styling)
- [ ] Testing

### Are there any known bugs?

- [x] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
* Add Router
* Add home-ish button-ish


### What ticket(s) do the changes resolve?
#17

Please Link Issues to this PR as necessary.
https://github.com/ConnorAndersonLarson/PlanetParty/issues/17#issue-855381014

### Notes for the Reviewer
Wrapping the planets in our Planetarium component in `<Link>`s added some annoying style side effects. Same with making the Planet Party! title a `<Link>`.


### How to Test Changes?
Confirm that click on a planet takes you to a new URL, that you can get back home by clicking on the title or on the back-ish button-ish, that actual planet images appear on the planet detail view, and that typing in a bad url (e.g. `/vulcan`) just takes you to the home page (but we can change that behavior if we want!)


### Next Iterations

knees4bees committed:
Set up router

knees4bees committed:
Get basic Router working without errors

knees4bees committed:
Add and debug Route for planet detail view

knees4bees committed:
Remove unused function

knees4bees committed:
Add Link to Planet components

knees4bees committed:
Change icon and title in browser tab

knees4bees committed:
Display all planet data in planet detail view

knees4bees committed:
Implement home-ish button-ish

knees4bees committed:
Clean up

knees4bees committed:
Add planet pics

knees4bees committed:
Update path for planet images

alexthompson207 commented on src/components/App/App.tsx:
```diff
@@ -36,11 +37,37 @@ class App extends React.Component<{}, AllData> {
+          )
+        }} />
+
+        <Route path="/:name" render={ ({ match }) => {
```
Should this be exact path?

alexthompson207 reviewed (approved):
This looks great! Really good work with this!

knees4bees committed:
Make gif a Link to home; add alt tags quote>
quote>
quote> Co-authored-by: ConnorAL connorandersonlarson@gmail.com
quote> Co-authored-by: AlexT ahthomps1@gmail.com
quote> Co-authored-by: KatieB 72777671+knees4bees@users.noreply.github.com

ConnorAndersonLarson reviewed (approved):
Looks good from our group walkthrough!

alexthompson207 committed:
Merge branch 'main' into feature/router

alexthompson207 commented:
Fixed merge conflicts, everything looks great!
